app/controllers/users_controller.rb

```diff
@@ -43,6 +43,12 @@ def confirm
     end
   end
 
+  def unconfirm
```

Comment:
This name is so weird. `unconfirm` is like i would like to remove the confirmation for this user but you are `reconfirming` actually. It should probably be `resend_confirmation`.